berkeley-function-call-leaderboard/eval_checker/eval_runner_helper.py

```diff
@@ -385,6 +385,12 @@
         "THUDM",
         "glm-4",
     ],
+    "yi-large-fc": [
+        "yi-large (FC)",
+        "https://platform.01.ai/",
```

Comment:
@HuanzhiMao @ShishirPatil The FC model does have function call doc in Chinese hosted on the [Chinese version website](https://platform.lingyiwanwu.com/docs#function-calling-%E5%B7%A5%E5%85%B7%E8%B0%83%E7%94%A8). The model and its doc are not hosted on the global website yet. I wonder if you prefer me to update the link to point to the Chinese website which doesn't have an English translation.

Reply:
Do you plan to have the model and its function doc up on the global website as well? If so, then I think we should use the global website URL here.

Reply:
This is indeed our global website URL, so no update is needed.

As @fantasist mentioned, we have already prepared the function call documentation for the Chinese website. Our production team is currently working on updating the documentation on the global website and expects to complete it within 1-2 days. We hope this does not block your testing procedure.

Thank you for checking!

Reply:
Awesome. Thanks.